code.js

```diff
@@ -1,3 +1,37 @@
+const { fun } = require("jsverify")
+
 function allPairsShortestPaths(graph) {
-    return [];
+    dist = make2d(graph.length)
+    for(i = 0; i < graph.length; i++){
+        dist[i][i] = 0
+    }
+    for(i = 0; i < graph.length; i++){
+        for(j = 0; j < graph[i].length; j++){
+            dist[graph[i][j][0]][i] = graph[i][j][1]
+        }
+    }
+    for(re = 0; re < 5; re++){//something can happen where some nodes dont actually get the shortest path, this fixes it
```

Comment:
with the graph that I was using sometimes a node would check to start looking and seeing if there was a way to get to a smaller node while it was could only get to nodes that had infinities. to solve this i had it repeat so that we didn't get any infinites as answers.

Reply:
That sounds like you have a bug in your implementation.